### PR TITLE
Verify that no neutron port is in failed state

### PR DESCRIPTION
**Pull request by dirkmueller**
If any of them is in failed state, something is wrong..

### CODE REVIEW

**Comment by aplanas** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3395,11 +3396,21 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # check that no port is in binding_failed state
+    for p in $(neutron port-list -f csv -c id --quote none | grep -v id); do
+        if neutron port-show $p -f value | grep -qx binding_failed; then
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: couldnt we move this test into neutron recipes, so that all cloud users would get notification of this error?

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: its a sanity check.. port bindings can still fail due to odd reasons, we just don't want it in our gating imho

**Comment by bmwiedemann** [comment]
+1

**Pull request merged**